lib/matplotlib/backends/backend_template.py

```diff
@@ -68,9 +69,9 @@
 import six
 
 import matplotlib
-from matplotlib._pylab_helpers import Gcf
 from matplotlib.backend_bases import RendererBase, GraphicsContextBase,\
-     FigureManagerBase, FigureCanvasBase
+     WindowBase, FigureCanvasBase, MainLoopBase, ToolbarBase
```

Comment:
take advantage of the fix to replace the backslash on the previous line by paretheses at the same time?